notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Configuration.scala

```diff
@@ -68,11 +68,18 @@ object Configuration {
     val config = fetchConfiguration("harvester")
     HarvesterConfiguration(
       jdbcConfig = jdbcConfig(config),
+      // currently this will ignore the CDK stack. To change to CDK, we should
+      // comment the next 5 lines, and uncomment the 5 lines following them.
```

Comment:
Will this be in it's own PR then?

Reply:
Yeah, I will immediately follow up with a new PR once everything looks good.